doc/source/whatsnew/v1.6.0.rst

```diff
@@ -182,7 +182,7 @@ MultiIndex
 
 I/O
 ^^^
--
+- Bug in :func:`read_sas` caused fragmentation of :class:`DataFrame` (:issue:`48595`)
```

Comment:
Could you mention here that ```:class:`.errors.PerformanceWarning` ``` is no longer raised as well?

Reply:
Done